deeplake/enterprise/dataloader.py

```diff
@@ -338,10 +352,10 @@ def __iter__(self):
         primary_tensor_name = self._primary_tensor_name
         buffer_size = self._buffer_size
 
-        tensors = self._tensors or map_tensor_keys(self.dataset, None)
-        dataset = dataset_to_libdeeplake(self.dataset)
+        tensors = self._tensors or map_tensor_keys(self.orig_dataset, None)
+        dataset = dataset_to_libdeeplake(self.orig_dataset)
```

Comment:
this won't account for queries I guess because those are applied on self.dataset